README.md

```diff
@@ -10,7 +10,7 @@
 
 ## Requirements
 
-ResponseDetective requires **iOS 8** or **OS X 10.10**.
+ResponseDetective requires **iOS 8+** or **OS X 10.10**.
```

Comment:
Let's be a little future-proof and add `+` to OS X version as well. And since we're here, I would also add information about Xcode version:


```md
ResponseDetective requires **Xcode 7.0+**, targeting **iOS 8.0+** or **OS X 10.10+**.
```